src/components/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { composeRenderProps } from 'react-aria-components';
```

Comment:
**question**: I assume this is temporary while working on the table?

Reply:
Yes, I just copied what you had which I believe referenced a function in utils that used this import.